alacritty_terminal/src/term/mod.rs

```diff
@@ -2050,6 +2063,21 @@ impl<T: EventListener> Handler for Term<T> {
         }
     }
 
+    #[inline]
+    fn set_prompt_mark(&mut self) {
+        // Don't mark prompts in alt-screen, since we likely not the one responsible for prompt
+        // marks handling anyway.
+        if self.mode.contains(TermMode::ALT_SCREEN) {
+            return;
+        }
```

Comment:
Why bother? We could just let them set it? It's not gonna have any negative effect, is it?